controllers/webhook_controller.py

```diff
@@ -16,56 +14,69 @@ class WebhookAdd(LoggedInHandler):
     def get(self):
         self._require_registration()
 
+        self.template_values['error'] = self.request.get('error')
+
         path = os.path.join(os.path.dirname(__file__), '../templates/webhook_add.html')
         self.response.out.write(template.render(path, self.template_values))
 
     def post(self):
         self._require_registration()
+        self._require_user()
+
+        # Name and URL must be non-None
+        url = self.request.get('url')
```

Comment:
Should we have get defaults in case these aren't supplied for some reason, or should we just let them throw errors?

Reply:
If you set the default to None, you'll redirect to the user friendlier error state.